app/util/conf/base_config_loader.py

```diff
@@ -87,6 +87,11 @@ def configure_defaults(self, conf):
         conf.set('git_askpass_exe', join(bin_dir, 'git_askpass.sh'))
         conf.set('git_ssh_exe', join(bin_dir, 'git_ssh.sh'))
 
+        # How slaves get the project
+        # Slaves would get the project from master if set to True. Otherwise it would just get the project in
+        # the same way how the master gets the project.
+        conf.set('get_project_from_master', True)
```

Comment:
Should we default to false? With this setting we should also be able to unskip the functional test `test_git_type_demo_project_config`. :smile: